deploy/shakenfist_ci/tests/test_artifacts.py

```diff
@@ -83,3 +83,76 @@ def test_resize_image_to_small(self):
             inst = self.test_client.get_instance(inst['uuid'])
 
         self.assertTrue(inst['state'] in ['creating-error', 'error'])
+
+    def test_artifact_ref_count_label(self):
+        url = ('https://sfcbr.shakenfist.com/gw-basic/gwbasic.qcow2')
+
+        img = self.system_client.cache_artifact(url)
+
+        # Get all artifacts once to make sure we get added to the list
+        image_urls = []
+        for image in self.system_client.get_artifacts():
+            image_urls.append(image['source_url'])
+        self.assertIn(url, image_urls)
+
+        # And then just lookup the single artifact
+        start_time = time.time()
+        found = False
+        while time.time() - start_time < 7 * 60:
+            img = self.system_client.get_artifact(img['uuid'])
+            if img['state'] == 'created':
+                found = True
+            time.sleep(5)
+
+        if not found:
+            self.fail('Image was not downloaded after seven minutes: %s' % (
+                img['uuid']))
+
+        self.assertIn('blobs', img)
+        self.assertEqual(len(img['blobs'], 1))
+        self.assertIn('1', img['blobs'])
+        self.assertIn('reference_count', img['blobs']['1'])
+        self.assertEqual(img['blobs']['1']['reference_count'], 1)
```

Comment:
Totally a nit, but in general asserts are written as (expected value, actual value) so they match the error message when they fail. That is, the "1" here should be the first argument.